asserts/device_asserts.go

```diff
@@ -44,6 +44,16 @@ func (mod *Model) Model() string {
 	return mod.HeaderString("model")
 }
 
+// ModelDisplayName returns the display-friendly name of the model or
+// falls back to Model if this was not set.
+func (mod *Model) ModelDisplayName() string {
+	display := mod.HeaderString("model-display-name")
```

Comment:
Can we please use ModelTitle/model-title for this one, or perhaps something else?

I'd just like to avoid the ambiguity of "name" in conversations, and would also not like to give the impression that every time we display the name it must necessarily be that field.

This pattern will probably end up being used everywhere else (we have a request for this on snaps already) so we should try to agree on the more general idea having that in mind.